script.js

```diff
@@ -5,10 +5,44 @@ $(function () {
   });
 });
 
-$('.navbar-nav .nav-link').click(function () {
-  $('.navbar-nav .nav-link').removeClass('active');
+$('.nav-link').click(function () {
```

Comment:
chyba są zbędne linijki 8-11

Reply:
nie wiem, to jeszcze Twoje były. Ja tam zabrałam z nich nieistniejącą klasę